.discourse-compatibility

```diff
@@ -0,0 +1 @@
+2.9.0.beta3: f8f5bacdf58a516c1a9000246587a7c09685f37e
```

Comment:
Btw just to check, this means that discourse-automation will only work with `f8f5bacdf58a516c1a9000246587a7c09685f37e`?

Reply:
Any Discourse version on 2.9.0.beta3 and below will be pinned to `f8f5bacdf58a516c1a9000246587a7c09685f37e` of this plugin.